itou/openid_connect/inclusion_connect/views.py

```diff
@@ -120,6 +120,7 @@ def inclusion_connect_resume_registration(request):
     # If there's a token in the session then the user already came back to the callback view : nothing to resume.
     # If there's no session then the user never started a registration on this device : nothing to resume either.
     if not ic_session or ic_session["token"]:
```

Comment:
Au cas où la session existe sans token ? Ou alors on veut que ça explose si le `token` n’a pas été défini avant, parce que ça ne devrait jamais arriver ?
```suggestion
    if not ic_session or ic_session.get("token"):
```

Reply:
Si pas de session, ou alors avec une session avec un token, on ne fait rien.
Si session sans token (aka token==none) on continue.